Update Lighthouse Docs (#202)
submariner-io/submariner-website: src/content/troubleshooting/_index.en.md

This is good time to familiarize yourself with [Service Discovery Architecture](../architecture/service-discovery/) if you haven't already.

##### Check ServiceExport for your Service

> For a Service to be accessible across clusters, you must first create a `ServiceExport` resource. Make sure the `ServiceExport` resource exists and has the same name and namespace as the Service you're trying to export. If it exists, check if the status of ServiceExport is "Exported"

Reviewer: Add that if it is not Exported, note the Status to know the reason why.

Author: Done.

> For a Service to be accessible across clusters, you must first create a `ServiceExport` resource. Make sure the `ServiceExport` resource exists and has the same name and namespace as the Service you're trying to export. If it exists, check if the status of ServiceExport is "Exported"

Reviewer: ```suggestion
For a Service to be accessible across clusters, you must first create a `ServiceExport` resource. Make sure the `ServiceExport` resource exists and has the same name and namespace as the Service you're trying to export. If it exists, check if the status of ServiceExport is `Exported`
```

> For a Service to be accessible across clusters, you must first export the Service via `subctl` which create a `ServiceExport` resource. Ensure the `ServiceExport` resource exists and check if its status condition indicates `Exported'. Otherwise, its status condition will indicate the reason it wasn't exported.

Reviewer 2: ```suggestion
For a Service to be accessible across clusters, you must first export the Service via `subctl` which creates a `ServiceExport` resource. Ensure the `ServiceExport` resource exists and check if its status condition indicates `Exported'. Otherwise, its status condition will indicate the reason it wasn't exported.
```

> For a Service to be accessible across clusters, you must first create a `ServiceExport` resource. Make sure the `ServiceExport` resource exists and has the same name and namespace as the Service you're trying to export. If it exists, check if the status of ServiceExport is `Exported'. If not, check the status and the message to know the exact reason.

Reviewer 2: ```suggestion
For a Service to be accessible across clusters, you must first export the Service via `subctl` which create a `ServiceExport` resource. Ensure the `ServiceExport` resource exists and check if its status condition indicates `Exported'. Otherwise, its status condition will indicate the reason it wasn't exported.
```

> ```kubectl get serviceexport -n <service-namespace> <service-name>```

Reviewer 3: Looking at https://github.com/submariner-io/lighthouse/pull/187
Can we also check the status of ServiceExport with with kubectl rollout or wait commands ?

Author: The below commands show you the status, but don't think rollout command supports it.
```
kubectl --context cluster2  describe serviceexport nginx
kubectl --context cluster2  get serviceexport nginx -o yaml
```
